src/components/news_item/NewsItem.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../core/styles/utilities";
+
+.newsitem {
+  width: 270px;
+  height: 320px;
+  border: 1px solid $diwala-detail-gray;
+  border-radius: 3px;
+  margin: 10px;
+  background-color: $diwala-background;
+  overflow-y: scroll;
+
+  &__section {
+    height: 50%;
+    padding: 30px;
+  }
+
+  &__logo {
+    margin: 0 auto;
+    text-align: center;
+    height: 100px;
+  }
+
+  &__quote {
+      text-align: left;
```

Comment:
To be picky, indentation on this whole block. What you can do is to download this if you are using atom, or something similar for your editor: https://github.com/sindresorhus/atom-editorconfig

Im using this for managing editor stuff: https://editorconfig.org/